internal/command/pause/worker.go

```diff
@@ -0,0 +1,72 @@
+package pause
+
+import (
+	"context"
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var wait uint64
+
+func newPauseVMCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "worker NAME",
+		Short: "Pause a worker",
+		RunE:  runPauseVM,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	command.PersistentFlags().Uint64Var(&wait, "wait", 0,
+		"wait the specified amount of seconds for the worker to stop running any VMs")
+
+	return command
+}
+
+func runPauseVM(cmd *cobra.Command, args []string) error {
```

Comment:
```suggestion
func runPauseWorker(cmd *cobra.Command, args []string) error {
```

Reply:
Renamed in https://github.com/cirruslabs/orchard/pull/73/commits/7fed7c2572967ec6143e86f9fd2fe712c5a1b41d.